contracts/PaymentSplitterFactory.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.3;
+
+import "./interfaces/IPaymentSplitterFactory.sol";
+import "./base/ProxyFactory.sol";
+import "./PaymentSplitter.sol";
+
+contract PaymentSplitterFactory is ProxyFactory, IPaymentSplitterFactory {
+    address internal _target;
+
+    constructor() {
+        PaymentSplitter target = new PaymentSplitter();
+        target.initialize("", new address[](0), new uint256[](0));
+        _target = address(target);
+    }
+
+    function deployPaymentSplitter(
+        address owner,
+        string calldata title,
+        address[] calldata payees,
+        uint256[] calldata shares
+    ) external override returns (address splitter) {
+        splitter = _createProxy(
```

Comment:
need parameters checking
1. owner is not address zero
2. titles length is not zero _ ?
etc.